jobs/kubelet/templates/config/gce.gcfg.erb

```diff
@@ -0,0 +1,5 @@
+<% if p("cloud-provider.type") == "gce" %> 
```

Comment:
That's a good idea. I'd like to avoid passing all of these parameters into so many jobs too, but it seems like adding a new job or using kubeconfig would not help with that. At least it would avoid the duplicate gce.gcfg files though